### ISSUE TITLE
RSS - wrong link to blog post detail (404)

### ISSUE THREAD

**Issue by Gappa** [pehapkari/pehapkari.cz#246]
The link in the RSS contains one more slash (`/`) after the `site_url`:

- https://pehapkari.cz/rss.xml
- https://github.com/pehapkari/pehapkari.cz/blob/master/templates/homepage/rss.xml.twig#L22

``` xml
<link>
https://pehapkari.cz//blog/2019/06/02/logic-belongs-to-entities/
</link>
```

Which returns a 404 on the web:

- https://pehapkari.cz//blog/2019/06/02/logic-belongs-to-entities/

Edit:

Guid & comments links are also broken:

- https://github.com/pehapkari/pehapkari.cz/blob/master/templates/homepage/rss.xml.twig#L24
- https://github.com/pehapkari/pehapkari.cz/blob/master/templates/homepage/rss.xml.twig#L27

**Comment by TomasVotruba** [comment]
Could you send PR?

**Comment by Gappa** [comment]
If the correct solution is to remove the slash from the twig template, I surely can :)

**Comment by TomasVotruba** [comment]
Resolved by #247
